### PR TITLE
add ceph backing store

### PR DESCRIPTION
**Pull request by orzhang**

### CODE REVIEW

File: pkg/scsi/backingstore/cephstore_linux.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2016 The GoStor Authors All rights reserved.
```
Comment (carmark) [inline pkg/scsi/backingstore/cephstore_linux.go]: 2016->2018?

File: pkg/scsi/backingstore/cephstore_linux.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+package backingstore
+
+import (
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/ceph/go-ceph/rados"
+	"github.com/ceph/go-ceph/rbd"
+	"github.com/gostor/gotgt/pkg/api"
+	"github.com/gostor/gotgt/pkg/scsi"
+	"strings"
```
Comment (carmark) [inline pkg/scsi/backingstore/cephstore_linux.go]: divide the import libraries  into three parts:
```
golang self packages
gostor packages
other packages
```
?

File: pkg/scsi/backingstore/cephstore_linux.go [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+func (bs *CephBackingStore) Open(dev *api.SCSILu, path string) error {
+
+	pathinfo := strings.SplitN(path, "/", 2)
+	if len(pathinfo) != 2 {
+		return fmt.Errorf("invalid device path string:%s", path)
+	}
+	poolName := pathinfo[0]
+	imageName := pathinfo[1]
+	fmt.Printf("path=%s\n", path)
```
Comment (carmark) [inline pkg/scsi/backingstore/cephstore_linux.go]: remove it or replace with logrus.Info?

File: pkg/scsi/backingstore/cephstore_linux.go (hunk 1 above)
Comment (orzhang) [reply pkg/scsi/backingstore/cephstore_linux.go]: done

File: pkg/scsi/backingstore/cephstore_linux.go (hunk 2 above)
Comment (orzhang) [reply pkg/scsi/backingstore/cephstore_linux.go]: done

File: pkg/scsi/backingstore/cephstore_linux.go (hunk 3 above)
Comment (orzhang) [reply pkg/scsi/backingstore/cephstore_linux.go]: done

**Comment by carmark** [comment]
LGTM

**Pull request merged**